docs/hazmat/primitives/cryptographic-hashes.rst

```diff
@@ -148,6 +148,37 @@ SHA-family of hashes.
 
     :raises ValueError: If the ``digest_size`` is invalid.
 
+SHA-3 family
+~~~~~~~~~~~~
+
```

Comment:
A couple of prose sentences about what the hell a SHA3 is would be good. it's secure, but slow, it's a new thing, probably you should just use it for compat where requierd, that sort of thing. with citations :-)

Reply:
SHA-3 is the most recent NIST secure hash algorithm standard. Despite the 
larger number SHA-3 is not considered to be better than SHA-2. Instead, it uses
a significantly different internal structure so that **if** an attack appears 
against SHA-2 it is unlikely to apply to SHA-3. SHA-3 is significantly slower 
than SHA-2 so at this time most users should choose SHA-2.